improver_tests/nbhood/circular_kernel/test_CircularNeighbourhood.py

```diff
@@ -77,11 +77,16 @@ class Test_apply_circular_kernel(IrisTest):
 
     """Test neighbourhood circular probabilities plugin."""
 
+    def setUp(self):
+        """Set up a cube."""
+
+        data = np.ones((1, 16, 16), dtype=np.float32)
+        self.cube = set_up_variable_cube(data, spatial_grid="equalarea",)
```

Comment:
It now occurs to me that we have a scalar dimension on these cubes (realization). This shouldn't occur in IMPROVER, so I thinkn we should change all these to remove scalar dimensions. It'll make all the tests slightly simpler too.